events/templates/events/events.html

```diff
@@ -1,74 +1,103 @@
-{% extends "events/base.html" %}
 {% load static %}
```

Comment:
I wonder why you choose to not use base.html but add the banner specifically in this file.

Reply:
No particular reason, 
I wanted to make a few changes and make the events html a proper html page and separate the css and js files.

Reply:
Got it!